[2022-09-01T23:57:06.729-0400] {plugins_manager.py:292} DEBUG - Plugins are already loaded. Skipping.
[2022-09-01T23:57:06.737-0400] {__init__.py:50} DEBUG - Loading core task runner: StandardTaskRunner
[2022-09-01T23:57:06.747-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-01T23:57:06.753-0400] {base_task_runner.py:70} DEBUG - Planning to run as the  user
[2022-09-01T23:57:06.757-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> from DB
[2022-09-01T23:57:06.766-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2022-09-01T23:57:06.766-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2022-09-01T23:57:06.772-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2022-09-01T23:57:06.772-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2022-09-01T23:57:06.772-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2022-09-01T23:57:06.772-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]>
[2022-09-01T23:57:06.773-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2022-09-01T23:57:06.773-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2022-09-01T23:57:06.773-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2022-09-01T23:57:06.778-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2022-09-01T23:57:06.778-0400] {taskinstance.py:1181} DEBUG - <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2022-09-01T23:57:06.780-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [queued]>
[2022-09-01T23:57:06.780-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-01T23:57:06.780-0400] {taskinstance.py:1369} INFO - Starting attempt 2 of 4
[2022-09-01T23:57:06.780-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-01T23:57:06.793-0400] {taskinstance.py:1389} INFO - Executing <Task(_PythonDecoratedOperator): hit_wikipedia_api> on 2022-09-02 03:49:13.195465+00:00
[2022-09-01T23:57:06.796-0400] {standard_task_runner.py:52} INFO - Started process 19676 to run task
[2022-09-01T23:57:06.806-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'test_etl', 'hit_wikipedia_api', 'manual__2022-09-02T03:49:13.195465+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/var/folders/2z/f_7z97sn6_33j1hnl38wlzs40000gn/T/tmpk19rzyzr', '--error-file', '/var/folders/2z/f_7z97sn6_33j1hnl38wlzs40000gn/T/tmpuq_85rmu']
[2022-09-01T23:57:06.808-0400] {standard_task_runner.py:80} INFO - Job 27: Subtask hit_wikipedia_api
[2022-09-01T23:57:06.813-0400] {cli_action_loggers.py:66} DEBUG - Calling callbacks: [<function default_action_log at 0x7fe1bcc393a0>]
[2022-09-01T23:57:06.829-0400] {settings.py:398} DEBUG - Disposing DB connection pool (PID 19676)
[2022-09-01T23:57:06.830-0400] {settings.py:260} DEBUG - Setting up DB connection pool (PID 19676)
[2022-09-01T23:57:06.831-0400] {settings.py:330} DEBUG - settings.prepare_engine_args(): Using NullPool
[2022-09-01T23:57:06.867-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-01T23:57:06.875-0400] {task_command.py:371} INFO - Running <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [running]> on host FAS-EALC-LIU-2
[2022-09-01T23:57:06.876-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [running]> from DB
[2022-09-01T23:57:06.897-0400] {taskinstance.py:1107} DEBUG - previous_execution_date was called
[2022-09-01T23:57:06.901-0400] {taskinstance.py:943} DEBUG - Clearing XCom data
[2022-09-01T23:57:06.938-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_etl
AIRFLOW_CTX_TASK_ID=hit_wikipedia_api
AIRFLOW_CTX_EXECUTION_DATE=2022-09-02T03:49:13.195465+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-02T03:49:13.195465+00:00
[2022-09-01T23:57:06.938-0400] {__init__.py:146} DEBUG - Preparing lineage inlets and outlets
[2022-09-01T23:57:06.939-0400] {__init__.py:190} DEBUG - inlets: [], outlets: []
[2022-09-01T23:57:07.052-0400] {python.py:173} INFO - Done. Returned value was: T. Rowe Price (id: 3990635, ns: 0)
[2022-09-01T23:57:07.062-0400] {__init__.py:107} DEBUG - Lineage called with inlets: [], outlets: []
[2022-09-01T23:57:07.062-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [running]> from DB
[2022-09-01T23:57:07.069-0400] {taskinstance.py:1418} DEBUG - Clearing next_method and next_kwargs.
[2022-09-01T23:57:07.070-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=test_etl, task_id=hit_wikipedia_api, execution_date=20220902T034913, start_date=20220902T035706, end_date=20220902T035707
[2022-09-01T23:57:07.071-0400] {taskinstance.py:2349} DEBUG - Task Duration set to 0.297263
[2022-09-01T23:57:07.079-0400] {cli_action_loggers.py:84} DEBUG - Calling callbacks: []
[2022-09-01T23:57:07.124-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-01T23:57:07.125-0400] {taskinstance.py:871} DEBUG - Refreshing TaskInstance <TaskInstance: test_etl.hit_wikipedia_api manual__2022-09-02T03:49:13.195465+00:00 [running]> from DB
[2022-09-01T23:57:07.147-0400] {dagrun.py:642} DEBUG - number of tis tasks for <DagRun test_etl @ 2022-09-02 03:49:13.195465+00:00: manual__2022-09-02T03:49:13.195465+00:00, state:running, queued_at: 2022-09-02 03:49:13.214385+00:00. externally triggered: True>: 3 task(s)
[2022-09-01T23:57:07.148-0400] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
